uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -0,0 +1,122 @@
+<div id="crm-contact-actions-list">
+  <div id="bootstrap-theme">
+    <div class="row">
+      <div class="col-md-4 contact_action_menu__user_info">
+        <div class="row">
+          <h3>User Informations:</h3>
+          <p><strong>User:</strong> <a href="#" class="text-primary"> 100 Lauren Ipsum </a></p>
```

Comment:
`strong` should be used. Please check all instances.

Reply:
Will do!